### DISCUSSION TITLE
is orientation() ignored by exportScene()?

### DISCUSSION THREAD

**Discussion by rawwerks** [google/model-viewer#3337] [Q&A]
I'm having issues where despite being sure to call updateFraming() after every time orientation() is called, I am not getting a WYSIWYG export of the .glb file. No matter what i do to orientation(), the rotation of the output GLB is unchanged. 

Does exportScene() take into account the current orientation of the scene? 

For example, below Left is inside the live modelviewer, center is the resulting .glb from exportScene() (following exactly the [example code](https://modelviewer.dev/examples/scenegraph/#exporter)) viewed on Win10 viewer, right is the resulting .glb viewed on modelviewer.dev/editor

![image](https://user-images.githubusercontent.com/19483938/161402115-7d776d50-cef9-4764-9444-4692c2f36ffe.png)

**Comment by rawwerks** [comment]
one idea that would fix my particular issue, is there an easy way to access the three.js property `eulerOrder`? https://stackoverflow.com/questions/9377396/how-to-use-quaternion-rotation-in-three-js-json-scene

**Reply by elalish** [reply]
Indeed, our exporter does not take orientation/scale into account when exporting. Perhaps it should? Would you like to make a PR? You can always hack your way down into our private `$scene` symbol, at which point you can access all of three.js.

**Reply by rawwerks** [reply]
> Indeed, our exporter does not take orientation/scale into account when exporting. Perhaps it should? Would you like to make a PR? You can always hack your way down into our private `$scene` symbol, at which point you can access all of three.js. 

@elalish - ok, thanks for the clarification. Could you show an example, gist, or documentation link regarding accessing `$scene`?

Is it as simple as `modelviewer.$scene.property` ?

**Reply by elalish** [reply]
We purposely don't document our internals. You'd need `getOwnSymbols`. If you don't know how, I'd recommend you avoid that route, as it is perilous.
